contribs/gmf/apps/desktop_alt/Controller.js

```diff
@@ -36,6 +36,7 @@ import gmfControllersAbstractDesktopController, {
 import appBase from '../appmodule.js';
 import gmfImportModule from 'gmf/import/module.js';
 import gmfFloorModule from 'gmf/floor/module.js';
+import gmfLidarprofileModule from 'gmf/lidarprofile/module.js';
```

Comment:
Missing`@import '~gmf/lidarprofile/lidarprofile.scss';`

Reply:
In lidarprofile we already have: `import './lidarprofile.scss';`...

Reply:
Okay... the style looks weird though...